testhelper/testhelper.go

```diff
@@ -383,6 +384,7 @@ func CreateSLAMService(
 		SensorValidationMaxTimeoutSecForTest,
 		SensorValidationIntervalSecForTest,
 		5*time.Second,
+		5*time.Second,
```

Comment:
should these be `15 * time.Minute`

Reply:
I felt for tests it was fine to leave it as a shorter timeout, but can increase if you prefer

Reply:
that makes sense to me. i am okay with either. maybe we just pass in the default variables so its the same as the real code?

Reply:
I'll make a set of constants for the tests to be consistent with the other constants in testhelper. otherwise I would have to export the defaults in the viam-cartographer package